### DISCUSSION TITLE
Additional sources

### DISCUSSION THREAD

**Discussion by MCas86** [3dproger/AxelChat#381] [Ideas]
I know the platforms aren't as prevalent but it would still be beneficial to include:

- Facebook
- Twitter

**Comment by 3dproger** [comment]
Hello! Thank you, I will try to integrate the application with these platforms in the future

**Comment by 3dproger** [comment]
Hi! I have good news, I added Facebook to AxelChat in 0.39.7. You can download it: https://github.com/3dproger/AxelChat/releases
